projects/ion/src/lib/table/table.component.spec.ts

```diff
@@ -164,6 +167,62 @@ describe('IonTableComponent', () => {
   });
 });
 
+describe('Table > columns header with tooltip', () => {
+  const events = jest.fn();
+  let columnHead: HTMLElement;
+  const columnsWithTooltip: Column[] = [
+    {
+      key: 'name',
+      label: 'Nome',
+      sort: true,
+      configTooltip: {
+        ionTooltipTitle: 'Eu sou um tooltip',
+      },
+    },
+    {
+      key: 'height',
+      label: 'Altura',
+      sort: true,
+    },
+  ];
+
+  const propsColumnWithTooltip: IonTableProps<Disco> = {
+    config: {
+      data,
+      columns: columnsWithTooltip,
+      pagination: {
+        total: 32,
+        itemsPerPage: 10,
+        page: 1,
+      },
+      loading: false,
+    },
+    events: {
+      emit: events,
+    } as SafeAny,
+  };
+
+  beforeEach(async () => {
+    await sut(propsColumnWithTooltip);
+  });
+
+  afterEach(() => {
+    fireEvent.mouseLeave(columnHead);
+  });
+
+  it('should render tooltip when it have a configTooltip', () => {
+    columnHead = screen.getByTestId('th-span-' + columnsWithTooltip[0].key);
+    fireEvent.mouseEnter(columnHead);
+    expect(screen.queryByTestId('ion-tooltip')).toBeInTheDocument();
```

Comment:
Previous comment could be applied here too.

Reply:
I agree, thanks for the review